### DISCUSSION TITLE
Annoying "Flashes" in Firefox's Toolbar UI Upon Binding Mode Activation

### DISCUSSION THREAD

**Discussion by anthony-S93** [i3/i3#5574] [Q&A] [answered]
I just upgraded to `firefox 115.0-1` and the first thing I noticed was the fact that the text in the toolbar (tab names, etc) now automatically greys out whenever the firefox window goes out of focus.  As in: 
![FirefoxUITextChangesColor](https://github.com/i3/i3/assets/69449791/f196d835-8a12-47c2-a95d-4ed27e70b000)

I believe this is what led to these annoying flashes I see on firefox's UI every time I activate a binding mode. 

![FirefoxUIFlashing](https://github.com/i3/i3/assets/69449791/991d5d60-d8c5-44c3-a296-0d2c16ea05c4)

This behavior was observed in all the gtk themes I've tried. I'm pretty sure this is an issue with Firefox since the issue went away after I downgraded to Firefox 114. I guess I'm just wondering if other `i3` users out there are able to reproduce this behavior.

**Comment by orestisfl** [comment]
I do get it but also with programs that bind keys without i3 and I also get the flashes in openbox, so it's quite safe to say this is on firefox

**Reply by anthony-S93** [reply, marked as answer]
@orestisfl Thanks for the reply. Yes. I came to the same conclusion after this post was made. I already filed a bug report with Mozilla.

**Reply by orestisfl** [reply]
Perfect, thank you.

If you could share the link in the mozilla bug tracker that would be great as well

**Reply by anthony-S93** [reply]
@orestisfl  Here's the link of the bugzilla report I made approximately two weeks ago: https://bugzilla.mozilla.org/show_bug.cgi?id=1841680

I honestly do not have the time to follow up with firefox's dev team over the past week, so it would be great if you could chip in.
